input/ig.xml

```diff
@@ -558,39 +558,39 @@
 				<reference value="Patient/ronny-irvine"/>
 			</reference>
 			<name value="Patient - Mr. Ronny Irvine (missing data for birth date)"/>
-			<description value="Mr. Ronny Irvine with missing data for birth date"/>
+			<description value="Shows a patient with missing data for birth date example for *AU Core Patient* profile."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-patient"/>
 		</resource>
 		<resource>
 			<reference>
 				<reference value="Patient/bennelong-anne"/>
 			</reference>
 			<name value="Patient - Mrs. Anne Bennelong"/>
-			<description value="Mrs. Anne Bennelong"/>
+			<description value="Shows an example of a patient with a Medicare Number and communication preferences for the *AU Core Patient* profile."/>
```

Comment:
Should say at least.

Shows an example of an Aboriginal patient with Medicare Number who prefers to communicate in Yugambal for *AU Core Patient* profile.